### DISCUSSION TITLE
Como empezar

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#50] [Q&A] [answered]
Buena tarde empece entregue katas este fin de semana y estoy viendo los vídeos de introducción alguien me podría guiar para poder empezar con las practicas

**Comment by doguedogue** [comment, marked as answer]
Los tres videos te van a dar el panorama general, luego al ver las practicas te vas a dar cuenta de lo que hay que ir haciendo, estamos en la fase digamos de análisis, diseño y planeación, todo es ponerle creatividad y ponerte en el rol.

**Reply by gersondlarosa** [reply]
muchas gracias ya voy a terminar el primer vídeo aun estoy confundido pero hay voy

**Comment by D-a-v-i-d-Vargas** [comment]
Hola a todos agradeceria si me pudieran ayudar, tuve un percance automivilistico con mi familia y no me pude conectar la ultima semana, apenas estoy viendo todos los videos, pero de verdad no tengo ninguna idea de como realizar las katas, y me quede mucho con el modelo de nuestra primera estapa que había mas instrucciones, solo quiero ponerme al corriente para entregar todo antes del día 4. 

p.d. he observado que esta parte se centra mas en investigar al cliente y como usaria nuestra pagina o la app, pero nuevamente no entiendo como hacer las katas o si necesitamos diseñar algo y apenas empece desde cero así que tengo miedo y estoy bastante confundido. 

Finalmente espero puedan ayudarme y gracias por leer.

**Reply by JuanSalvador73** [reply]
Primero, hay que leer con cuidado toda la info del Intro a FrontEnd. Los documentos que están en las prácticas son para eso. Ya una vez que uno hizo eso, en base a esos mismos documentos te basas para hacer los tuyos o usas esos mismos, eso si, con tu propia info y los redactas  acorde a tu trabajo. Muchos aquí han hecho eso mismo aquí - me incluyo - y han publicado su trabajo, checa las discusiones y también los videos de las sesiones en vivo. 

Básicamente esta etapa de trabajo es investigación, y creación de maquetas de como quedaría el sitio online, todavía no se ha visto en forma como trabajar en código - salvo la intro del jueves pasado - para concretarlo.
